Source/WebCore/html/PluginDocument.cpp

```diff
@@ -107,11 +107,6 @@ void PluginDocumentParser::createDocumentStructure()
     if (document.frame())
         document.frame()->injectUserScripts(UserScriptInjectionTime::DocumentStart);
 
-#if PLATFORM(IOS_FAMILY)
-    // Should not be able to zoom into standalone plug-in documents.
-    document.processViewport("user-scalable=no"_s, ViewportArguments::Type::PluginDocument);
```

Comment:
I think this is the only use of `Type::PluginDocument`. Maybe we can want to get rid of this case. We don't base any logic on this case anyways it seems

Reply:
ooh, looking